Framework/API/inc/MantidAPI/Column.h

```diff
@@ -163,6 +163,14 @@ class MANTID_API_DLL Column {
     return vec;
   }
 
+  virtual bool equals(Column *, double) const {
```

Comment:
Can we make this signature `const Column &` rather than a non-const pointer? I think that means you can do away with the const-cast below.